pkgs/by-name/li/libu2f-emu/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  autoconf-archive,
+  pkg-config,
+  openssl,
+  gtest,
+  gitUpdater,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libu2f-emu";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "eveldun";
+    repo = "libu2f-emu";
+    tag = version;
+    hash = "sha256-AzH1rZFqEH8sovZZfJykvsEmCedEZWigQFHWHl6/PdE=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    autoconf-archive
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl.dev
+    gtest
+  ];
+
+  env.NIX_CFLAGS_COMPILE = "-Wno-error=deprecated-declarations";
```

Comment:
This would ideally be reported upstream (or a patch sent)

Reply:
The upstream has issues disabled, and I don't know enough openssl to fix it myself. Maybe I can ping @eveldun like this?

The issue is that openssl3 deprecated some low-level functions that the project uses.